modules/interactive_music/doc_classes/AudioStreamInteractive.xml

```diff
@@ -198,6 +198,9 @@
 		<constant name="TRANSITION_TO_TIME_START" value="1" enum="TransitionToTime">
 			Transition to the start of the destination clip.
 		</constant>
+		<constant name="TRANSITION_TO_TIME_PREVIOUS_POSITION" value="2" enum="TransitionToTime">
+			Transition to the previous position in the destination clip. For example, if clip [code]a[/code] is at position [code]3.5s[/code] when the stream transitions to clip [code]b[/code], then a later transition back to clip [code]a[/code] using this mode will resume playback of [code]a[/code] from [code]3.5s[/code]. If the destination clip has not previously played, it will play from the start.
```

Comment:
```suggestion
			Transition to the last played position in the destination clip, if there was a previous transition from that clip. Otherwise, plays from the start of the destination clip.
```
How about this?